src/Stratis.Bitcoin/Builder/FullNodeFeatureExecutor.cs

```diff
@@ -57,11 +57,11 @@ public void Initialize()
 
                 this.Execute(feature =>
                 {
-                    this.signals.Publish(new FullNodeEvent() { Message = $"Initializing feature '{feature.GetType().Name}'" });
+                    this.signals.Publish(new FullNodeEvent() { Message = $"Initializing feature '{feature.GetType().Name}'.", State = FullNodeState.Initializing.ToString() });
                     feature.State = FeatureInitializationState.Initializing;
                     feature.InitializeAsync().GetAwaiter().GetResult();
                     feature.State = FeatureInitializationState.Initialized;
-                    this.signals.Publish(new FullNodeEvent() { Message = $"Feature '{feature.GetType().Name}' initialized" });
+                    this.signals.Publish(new FullNodeEvent() { Message = $"Feature '{feature.GetType().Name}' initialized.", State = FullNodeState.Initializing.ToString() });
```

Comment:
Is this meant to be `Initializing` as well?

Reply:
Yes, it is a bit confusing as the event is for the state of the full node as a whole, not the individual feature.